model/src/main/java/com/epam/esm/mapper/GiftCertificateTagRowMapper.java

```diff
@@ -17,6 +17,6 @@ public GiftCertificateTag mapRow(ResultSet resultSet, int i) throws SQLException
         BigInteger idGiftCertificate = new BigInteger(resultSet.getString("id_gift_certificate"));
         BigInteger idTag = new BigInteger(resultSet.getString("id_tag"));
 
-        return new GiftCertificateTag(id, idGiftCertificate, idTag);
+        return new GiftCertificateTag();
```

Comment:
It's suspicious to return empty created GiftCertificateTag. pls, double check and explain the purpose of such actions